src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -59,7 +62,7 @@ public class TestJsonCommand : PSCmdlet
         /// <returns>Return value is unreachable since we always rethrow.</returns>
         private static bool UnwrapException(Exception e)
         {
-            if (e is TargetInvocationException)
+            if (e is TargetInvocationException && e.InnerException != null)
             {
                 ExceptionDispatchInfo.Capture(e.InnerException).Throw();
```

Comment:
<https://github.com/dotnet/runtime/issues/20532> added `ExceptionDispatchInfo.Throw(Exception)`, which would simplify this a bit. No need to make such a change, of course.

Reply:
Thanks! Fixed.